Sources/SKCore/FallbackBuildSystem.swift

```diff
@@ -51,15 +51,18 @@ public final class FallbackBuildSystem {
 
   public var indexPrefixMappings: [PathPrefixMapping] { return [] }
 
-  public func buildSettings(for uri: DocumentURI, language: Language) -> FileBuildSettings? {
+  public func buildSettings(for uri: DocumentURI, language: Language, isFallback: Bool = true) -> FileBuildSettings? {
+    var fileBuildSettings: FileBuildSettings?
```

Comment:
Instead of passing `isFallback` here, I would prefer to set it to `true` here, since it is coming from a fallback build system and overriding it to `false` in `BuildSystem` if we don’t have a real build system.

Reply:
right, by doing so, the func buildSettings in FallbackBuildSystem would be more clean.